### PR TITLE
Styling: soften some borders into box shadows

### PR DESCRIPTION
**Pull request by alex-sherman**
Just had some ideas for how to soften some borders, no need to merge this if you're not into it. Feel free to revert whatever parts, or cherry pick only sections.

### CODE REVIEW

File: src/components/Pages/Board/Views/index.scss [hunk 1]
```diff
@@ -8,10 +8,12 @@
   flex-direction: column;
   flex-grow: 1;
   .views-group-name {
-    border: 1px solid #d4d6d6;
+    background-color: #eef1f5;
+    z-index: 2;
+    //border-right: 1px solid #d4d6d6;
```
Comment (DarrenZLew) [inline src/components/Pages/Board/Views/index.scss]: Can you delete these comment?

File: src/components/UI/BoardComponents/Card/index.scss [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-    transition: background-color 0.8s ease;
-    .edit-icon {
-      visibility: visible;
-      opacity: 1;
-      transition: opacity 0.8s linear;
-    }
-    box-shadow: 2px 3px 6px rgba(0, 0, 0, 0.26), 2px 3px 6px rgba(0, 0, 0, 0.33);
+    transition: background-color, box-shadow 0.35s ease;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.26);
+    z-index: 10;
```
Comment (DarrenZLew) [inline src/components/UI/BoardComponents/Card/index.scss]: What's the point of the z-index?

File: src/components/UI/Navbar/components/NavbarList/index.scss [hunk 3]
```diff
@@ -1,15 +1,18 @@
+#navbar {
+  box-shadow: 0px 0px 13px 0px rgba(98, 98, 102, 0.16);
+  z-index: 1;
+  //border-bottom: 1px solid #d4d6d6;
```
Comment (DarrenZLew) [inline src/components/UI/Navbar/components/NavbarList/index.scss]: Can you delete this comment?

**Comment by DarrenZLew** [comment]
Looks good overall

File: src/components/UI/BoardComponents/Card/index.scss (hunk 2 above)
Comment (alex-sherman) [reply src/components/UI/BoardComponents/Card/index.scss]: This was supposed to make the shadow cover the adjacent cards but apparently did not. I fixed it by using some `:before` shenanigans in the latest commit.

**Review by DarrenZLew — APPROVED** [review]

**Pull request merged**